vcbuild.bat

```diff
@@ -172,8 +181,6 @@ if "%target%"=="Clean" echo deleting %~dp0deps\icu
 if "%target%"=="Clean" rmdir /S /Q %~dp0deps\icu
 :no-depsicu
 
-call :getnodeversion || exit /b 1
```

Comment:
The next line uses `FULLVERSION` which is populated in `getnodeversion`.

Reply:
Fixed by making the rmdir line more generic.

Reply:
%TAG% is used in `project-gen` too.